workspaces/Cargo.toml

```diff
@@ -31,6 +31,7 @@ tokio-retry = "0.3"
 tracing = "0.1"
 url = { version = "2.2.2", features = ["serde"] }
 
+near-abi-client = { git = "https://github.com/shariffdev/near-abi-client-rs", branch = "chore/bump-workspaces" }
```

Comment:
near-abi-client 0.1.1 is now released, let's update: https://github.com/near/near-abi-client-rs/releases/tag/near-abi-client-v0.1.1